ui/app/AppLayouts/Wallet/WalletUtils.qml

```diff
@@ -346,7 +356,8 @@ QtObject {
             catch (e) {
                 return ""
             }
-
+        case Constants.routerErrorCodes.router.errLowAmountInForHopBridge:
+            return qsTr("bonder fee greater than estimated received, a higher amount is needed to cover fees")
```

Comment:
```suggestion
            return qsTr("bonder fee greater than estimation received, a higher amount is needed to cover fees")
```

?

Reply:
That's something I took from the hop service:

<img width="548" alt="image" src="https://github.com/user-attachments/assets/a7fec986-05a7-4615-b455-04a472b5c5d5">